backend/howtheyvote/cli/pipeline.py

```diff
@@ -129,3 +130,12 @@ def sessions(term: int) -> None:
     and locations."""
     pipeline = SessionsPipeline(term=term)
     pipeline.run()
+
+
+@pipeline.command()
+@click.option("--date", type=click.DateTime(formats=["%Y-%m-%d"]), required=False)
+def oeil_summaries(date: datetime.date | None = None) -> None:
+    """Scrape all OEIL summaries for texts voted on in the last four weeks.
+    Does not take into account any existing summaries nor ids."""
```

Comment:
Note to self: Is this what it actually does? Or does it scrape summaries for votes on the given day.